tests/namespaced-admission-policy-tests.bats

```diff
@@ -22,22 +24,21 @@ setup_file() {
 }
 
 @test  "[AdmissionPolicy tests] Launch a pod which does not violate privileged pod policy" {
-	kubectl_apply_should_succeed $RESOURCES_DIR/not-violate-privileged-pod-policy.yaml
+	kubectl_apply $RESOURCES_DIR/not-violate-privileged-pod-policy.yaml
 }
 
 @test  "[AdmissionPolicy tests] Update privileged pod policy to check only UPDATE operations" {
-        run kubectl --context $CLUSTER_CONTEXT patch admissionpolicy privileged-pods --type=json -p ' [{ "op": "remove", "path": "/spec/rules/0/operations/1" }, { "op": "replace", "path": "/spec/rules/0/operations/0", "value": "UPDATE" } ]'
-	[ "$status" -eq 0 ]
+        kubectl patch admissionpolicy privileged-pods --type=json -p ' [{ "op": "remove", "path": "/spec/rules/0/operations/1" }, { "op": "replace", "path": "/spec/rules/0/operations/0", "value": "UPDATE" } ]'
```

Comment:
shouldn't we still be checking the retcode after the kubectl patch?

Reply:
When you prepend command with `run` it puts it's exit code to `$status` variable and returns 0, then you have to check it manually.
If you call command directly and it fails then bash errexit is triggered automatically.
So wrapping command in `run` and check exit code is just a wrapper that brings no value.

Reply:
I see, thanks! resolving